ailab/ailab/views.py

```diff
@@ -0,0 +1,5 @@
+from django.shortcuts import redirect
```

Comment:
Please add the follow line above this line:

from __future__ import absolute_import, division, print_function, unicode_literals